packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -6239,38 +6239,28 @@ describe('plugin-meetings', () => {
           }
         );
 
-        it('canUseVoip is enabled based on locus info when the conditions are met', () => {
+        it('canUseVoip is disabled based on locus info when the required policies are missing', () => {
```

Comment:
not based on locus info

Reply:
you are correct. I've fixed this. Interestingly though, I didn't actually make that change this time round. That phrase was there before, but the diff makes it look like I wrote it in this PR